### PR TITLE
Added Basic README.md File

### PR DESCRIPTION
**Pull request by SakuraBlossomTree**
This is the README.md file 

It is a basic README

Please review and let me know if you want any changes to this

### CODE REVIEW

**Comment by Prakash4844** [comment]
Hey, sorry but this need editing. and i don't think adding direct command in readme.md is a good idea. we will be working on docs as soon as a Minimal viable product is ready. 

in the mean time just add Project Name aligned center and a little description for it. 

Tomorrow, i will try to work out a logo. and provide you that so you can add it.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-# Name-subject-to-change
-{NAME} - A Distributed VCS in Python. A distributed revision control system in Python. an Open Source project covered by the GNU General Public License.
+# TIT
+{TIT} - A Distributed VCS in Python. A distributed revision control system in Python. an Open Source project covered by the GNU General Public License.
```
Comment (Prakash4844) [inline README.md:2]: this should be like this

TIT - A Distributed Version Control System in Python. an Open Source project covered by the GNU General Public License.

Comment (SakuraBlossomTree) [reply README.md:2]: okay sure 

I will edit this

**Comment by SakuraBlossomTree** [comment]
I have edited the README file again 

Keep a message

**Comment by Prakash4844** [comment]
Looks good to me. Mergin. 

Thanks for your contribution.

**Comment by SakuraBlossomTree** [comment]
Thanks alot

**Pull request merged**